docs/tutorial.md

```diff
@@ -603,6 +602,11 @@ These loops are possible because `Object.values(obj)` and `Object.keys(obj)` bot
 `Object.entries(obj)` returns an array where each item is an array with 2 items: a key and its corresponding value.  
 It's important to know that, unlike the `Object.values()` and `Object.keys()` functions, the `Object.entries()` function does not guarantee that the order is the same as the order when the object was defined.
 
+There is also a `for in` loop. However, you will most often want to use `for of` instead of `for in` because there are key differences.  
+The `for in` loop loops over the keys of an object instead of the values. (The index incase it is an array)
```

Comment:
in case

`incase` is a verb (technically misspelling of `encase`) and means `to enclosing something`